MashUp-iOS/MashUp-iOS/Source/Screen/Seminar Schedule/View/AttendenceTimeAndCheckView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  AttendenceTimeAndCheckView.swift
+//  MashUp-iOS
+//
+//  Created by 이문정 on 2022/06/18.
+//  Copyright © 2022 Mash Up Corp. All rights reserved.
+//
+
+import SnapKit
+import Then
+import UIKit
+import MashUp_Core
+
+class AttendenceTimeAndCheckView: UIView {
```

Comment:
```suggestion
final class AttendenceTimeAndCheckView: UIView {
```
final로 정의되면 좋을 것 같습니답